src/ChangeStream.php

```diff
@@ -130,6 +139,15 @@ public function rewind()
                 $this->hasAdvanced = true;
                 $this->resumeToken = $this->extractResumeToken($this->csIt->current());
             }
+            // If the cursorId is 0, the server has invalidated the cursor so we
+            // will never perform another getMore. This means that we cannot
+            // resume and we can therefore unset the resumeCallable, which will
+            // free any reference to Watch. This will also free the only
+            // reference to an implicit session, since any such reference
+            // belongs to Watch.
```

Comment:
I propose pointing to the earlier extended comment for `next()` rather than reproducing the full explanation:

```
// As with next(), free the callable once we know it will never be used
```